op-dispute-mon/flags/flags.go

```diff
@@ -0,0 +1,96 @@
+package flags
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli/v2"
+
+	"github.com/ethereum-optimism/optimism/op-dispute-mon/config"
+	opservice "github.com/ethereum-optimism/optimism/op-service"
+	oplog "github.com/ethereum-optimism/optimism/op-service/log"
+	opmetrics "github.com/ethereum-optimism/optimism/op-service/metrics"
+	"github.com/ethereum-optimism/optimism/op-service/oppprof"
+	"github.com/ethereum-optimism/optimism/op-service/txmgr"
+)
+
+const (
+	envVarPrefix = "OP_DISPUTE_MON"
+)
+
+func prefixEnvVars(name string) []string {
+	return opservice.PrefixEnvVar(envVarPrefix, name)
+}
+
+var (
+	// Required Flags
+	L1EthRpcFlag = &cli.StringFlag{
+		Name:    "l1-eth-rpc",
+		Usage:   "HTTP provider URL for L1.",
+		EnvVars: prefixEnvVars("L1_ETH_RPC"),
+	}
+	// TODO(client-pod#531): Add Required Flags
```

Comment:
Probably don't need these todos.  The aim of the first issue is just to get metrics and pprof up and running (and built/tested in CI).  The rest of the CLI args will all come in as part of the stories that use them.  We're aiming for vertical slices of functionality.

We've got the L1EthRpcFlag and will need it so fine to leave that it but aim for doing one small step at a time.  Once we get the initial minimal framework merged we can parallelise some of the work.

Reply:
Perfect :)